src/Compilers/CSharp/Portable/DocumentationComments/DocumentationCommentIDVisitor.PartVisitor.cs

```diff
@@ -170,6 +170,11 @@ public override object VisitTypeParameter(TypeParameterSymbol symbol, StringBuil
 
             public override object VisitNamedType(NamedTypeSymbol symbol, StringBuilder builder)
             {
+                if (symbol.IsTupleType)
+                {
+                    return VisitTupleType((TupleTypeSymbol)symbol, builder);
```

Comment:
Is this equivalent to `VisitNamedType(((TupleTypeSymbol)symbol).UnderlyingType, builder)`?